main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/dequelabs/nav-json-validator/navjson"
+)
+
+var silent *bool
+var file *string
+
+func init() {
+	silent = flag.Bool("silent", false, "Silence output")
+	file = flag.String("file", "docs/nav.json", "Path to nav.json file")
+	flag.Parse()
+}
+
+func check(err error) {
+	if err == nil {
+		return
+	}
+
+	if *silent {
+		os.Exit(1)
+	}
+
+	panic(err)
+}
+
+func main() {
+	data, err := ioutil.ReadFile(*file)
+	check(err)
+
+	_, err = navjson.Parse(string(data))
```

Comment:
ha ha, give `_` a better name :). You once did this to me in a different PR. The world is a circle :)

Reply:
It's a Go thing. It means "immediately dispose of this".